src/components/organisms/Nav.js

```diff
@@ -132,14 +184,42 @@ const Nav = () => {
             confirmText="Sign Up"
             cancelHandler={() => setModalState(modalStates.closed)}
           >
+            <label htmlFor="name">
+              Display Name
+              <input
+                type="text"
+                name="name"
+                value={isUsername}
```

Comment:
once you change the names of the states as I said above, make sure you refactor the references

Reply:
got it!

Reply:
what are the names being changed to?

Reply:
oh nvm i see